src/Components/Pages/PortfolioSize/PortfolioSize.tsx

```diff
@@ -257,27 +276,77 @@ export const AcrisLinks: React.FC<ACRISLinksProps> = ({
   );
 };
 
+const FindOtherBuildings: React.FC = () => {
+  return (
+    <div className="callout-box">
+      <strong>To find other buildings your landlord might own, you can:</strong>
+      <ul>
+        <li>
+          Google your landlord’s name and find mentions of other buildings they
+          may own in the press.
+        </li>
+        <li>
+          Talk to your neighbors to see if they know of any other buildings your
+          landlord may own.
+        </li>
+      </ul>
+    </div>
+  );
+};
+
 export const AcrisAccordions: React.FC<BuildingData> = (props) => {
-  const MAX_PROPERTIES = 5;
+  const INIT_DISPLAY = 5;
+  const LOAD_MORE_AMOUNT = 5;
+
   // TODO: decide how to handle these cases, for now exclude. might also want to exclude if no acris_docs, but for now leave in.
   const relatedProperties = props.related_properties
     ?.filter((bldg) => bldg.unitsres > 0)
-    .sort(prioritizeBldgs)
-    .slice(0, MAX_PROPERTIES);
+    .sort(prioritizeBldgs);
+  const totalCount = relatedProperties?.length;
+  const [visibleCount, setVisibleCount] = useState(INIT_DISPLAY);
+  const [showLoadMoreButton, setShowLoadMoreButton] = useState(
+    totalCount ? totalCount > INIT_DISPLAY : false
+  );
+
+  const loadMoreBuildings = () => {
+    if (!!totalCount && visibleCount + LOAD_MORE_AMOUNT >= totalCount) {
+      setVisibleCount(totalCount);
+      setShowLoadMoreButton(false);
+    } else {
+      setVisibleCount(visibleCount + LOAD_MORE_AMOUNT);
+    }
+  };
 
   return (
-    <ul>
-      {relatedProperties?.map((bldg, i) => {
-        return (
+    <>
+      <ul>
+        {relatedProperties?.slice(0, visibleCount).map((bldg, i) => (
           <li key={i}>
             <AcrisLinks
               {...bldg}
               info="You only need to find your building’s landlord’s signature on one of the documents below"
               accordion
             />
           </li>
-        );
-      })}
-    </ul>
+        ))}
+      </ul>
+      <div className="related-buildings__footer">
+        Showing {visibleCount} of {totalCount} buildings
```

Comment:
Steve found that variables in the jsx messes up the screen reader, but works fine with strings
```suggestion
        {`Showing ${visibleCount} of ${totalCount} buildings`}
```